### PR TITLE
[ZEPPELIN-5962] LivyInterpreter support more parameter when create session

### PR DESCRIPTION
**Pull request by zhaomoran**
### What is this PR for?
LivyInterpreter support more parameter when create session

### What type of PR is it?
Improvement

### Todos
* [x] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/projects/ZEPPELIN/issues/ZEPPELIN-5962

### How should this be tested?
* Strongly recommended: add automated unit tests for any new or changed behavior
* Outline any manual steps to test the PR here.

### Questions:
* Does the license files need to update?
* Is there breaking changes for older versions?
* Does this needs documentation?

### CODE REVIEW

File: livy/src/main/java/org/apache/zeppelin/livy/BaseLivyInterpreter.java [hunk 1]
```diff
@@ -776,15 +780,22 @@ private static class CreateSessionRequest {
...
+    CreateSessionRequest(String kind, String user, Map<String, String> conf, Map<String, String> params) {
       this.kind = kind;
       this.user = user;
       this.conf = conf;
+      this.params = params;
     }
 
     public String toJson() {
-      return gson.toJson(this);
+      JsonObject jsonObject = new JsonObject();
```
Comment (jongyoul) [inline livy/src/main/java/org/apache/zeppelin/livy/BaseLivyInterpreter.java:793]: You might miss `import` statement for `JsonObject`?

**Comment by jongyoul** [comment]
@zhaomoran Hello, thank you for your contribution. I left a comment and could you please add test cases? I read your ticket and looks fine but I think it would be better to have a test for this case.

**Comment by zhaomoran** [comment]
@jongyoul OK，I just presented the idea here. If the community thinks it's a good idea, I'll add test cases to it.

**Comment by jongyoul** [comment]
I believe that your contribution must help others. :-)

**Comment by Reamer** [comment]
Please perform a rebase. I have recently migrated the tests to JUnit5. Currently there are merge conflicts.

**Comment by zhaomoran** [comment]
@jongyoul This is my first time submitting a PR. Is there anything else I need to do?

**Comment by jongyoul** [comment]
@zhaomoran The last step is making all CI green. We have some flaky tests but I usually try to execute some failed tests again to make it green. Your contribution itself is proper to be merged. Could you please wait for a while until the CI becomes green?

**Review by Reamer — APPROVED** [review]
lgtm

**Comment by jongyoul** [comment]
@zhaomoran Thank you for your first contribution. I hope you contribute more and more. I merged your PR and assigned you with the JIRA ticket as well.

**Pull request merged**